Framework/src/Triggers.cxx

```diff
@@ -119,9 +120,19 @@ TriggerFcn Never(const Activity& activity)
   };
 }
 
-TriggerFcn EndOfRun(const Activity&)
+TriggerFcn EndOfRun(const std::string& kafkaBrokers, const std::string& topic, const Activity& activity)
 {
-  return NotImplemented("EndOfRun");
+  auto returnedActivity = activity;
+  auto poller = std::make_shared<core::KafkaPoller>(kafkaBrokers, "EOR_postprocessing_group");
```

Comment:
Forgive my lack of familiarity with kafka here...
What are the implications of using a common group ID for all postprocessing tasks who want to trigger on EOR? Does it mean that just one will get the event?